src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -771,14 +771,21 @@ private synchronized void update()
             // creating local final variables and pass that to the lambda function
             // in order to avoid that.
             final String endpointID = trackBitrateAllocation.endpointID;
+
+            final Endpoint endpointOwningTrack = destinationEndpoint.getConference().getLocalEndpoint(endpointID);
```

Comment:
We'll have to discuss if we want to fix this or not (as you've seen, quite a few things assuming the same payload types), but if we do do this, we should expose the payload types a different way, I think.  The first thing that comes to mind is having the `Transceiver` expose a read-only version of its `StreamInformationStore` (which we could then expose at the `Endpoint` level)

Reply:
See https://github.com/jitsi/jitsi-media-transform/blob/master/src/main/kotlin/org/jitsi/nlj/Transceiver.kt#L75